### PR TITLE
Addresses BACKLOG-40066: New schedule execution permission

### PR DESCRIPTION
**Pull request by mbrasil**
@pentaho/millenniumfalcon please review

### CODE REVIEW

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1h 5m 2s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl \
assemblies/pentaho-solutions,extensions,repository,user-console
```
---
:ok_hand: All tests passed!
 
Tests run: **2005**, Failures: **0**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5560/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: user-console/src/main/java/org/pentaho/mantle/client/ui/PerspectiveManager.java [hunk 1]
```diff
@@ -290,9 +315,8 @@ private void loadResourceBundle( final HasText textWidget, final IPluginPerspect
       } );
 
     } catch ( Exception e ) {
-      String msg = "Error loading message bundle: " + e.getMessage();
-      log.error( msg, e );
-      Window.alert( msg );
+      Window.alert( "Error loading message bundle: " + e.getMessage() );
+      e.printStackTrace();
```
Comment (afrjorge) [inline user-console/src/main/java/org/pentaho/mantle/client/ui/PerspectiveManager.java:319]: Why remove the logger?

Comment (mbrasil) [reply user-console/src/main/java/org/pentaho/mantle/client/ui/PerspectiveManager.java:319]: At first, I made that change but later realized that I must include this in GWT inherits and didn't find a way. So I left it as it were.

**Review by afrjorge — APPROVED** [review]

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :warning: Build finished in 1h 1m 57s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl \
assemblies/pentaho-solutions,extensions,repository
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 11 test(s) failed:

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunGetWithParameter</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunGetErrorResponse</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunPost</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunGetAxisServletHooks</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunGetThrowError</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunGet</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunPut</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.test.platform.plugin.services.webservices.AxisServiceWsdlGeneratorTest.testBadInit3</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.test.platform.plugin.services.webservices.WsdlPageTest.testMissingPathParamProvider</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.test.platform.plugin.services.webservices.WsdlPageTest.testMissingServiceName</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.test.platform.plugin.services.webservices.WsdlPageTest.testBadServiceName</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

  
Tests run: **2005**, Failures: **11**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5560/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :warning: Build finished in 1h 1m 25s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl \
assemblies/pentaho-solutions,extensions,repository
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 11 test(s) failed:

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunGetWithParameter</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunGetErrorResponse</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunPost</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunGetAxisServletHooks</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunGetThrowError</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunGet</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunPut</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.test.platform.plugin.services.webservices.AxisServiceWsdlGeneratorTest.testBadInit3</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.test.platform.plugin.services.webservices.WsdlPageTest.testMissingPathParamProvider</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.test.platform.plugin.services.webservices.WsdlPageTest.testMissingServiceName</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.test.platform.plugin.services.webservices.WsdlPageTest.testBadServiceName</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

  
Tests run: **2005**, Failures: **11**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5560/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by Pancho7 — APPROVED** [review]

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :warning: Build finished in 1h 6m 27s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl \
assemblies/pentaho-solutions,extensions,repository
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 11 test(s) failed:

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunGetWithParameter</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunGetErrorResponse</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunPost</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunGetAxisServletHooks</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunGetThrowError</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunGet</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.platform.plugin.services.webservices.content.AxisServiceExecutorTest.testRunPut</b> (click to expand)</summary><p>

```
Exception occurred
```

</p></details>

<details><summary><b>org.pentaho.test.platform.plugin.services.webservices.AxisServiceWsdlGeneratorTest.testBadInit3</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.test.platform.plugin.services.webservices.WsdlPageTest.testMissingPathParamProvider</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.test.platform.plugin.services.webservices.WsdlPageTest.testMissingServiceName</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

<details><summary><b>org.pentaho.test.platform.plugin.services.webservices.WsdlPageTest.testBadServiceName</b> (click to expand)</summary><p>

```
${result.errorDetails}
```

</p></details>

  
Tests run: **2005**, Failures: **11**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5560/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 1h 56s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl \
assemblies/pentaho-solutions,extensions,repository
```
---
:ok_hand: All tests passed!
 
Tests run: **2005**, Failures: **0**, Skipped: **5** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/platform/job/PR-5560/5/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**